.gitignore

```diff
@@ -32,3 +32,6 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+*.lock.hcl
```

Comment:
`.terraform.lock.hcl` がバージョン管理の対象外になりそうなのですが、意図はあるのでしょうか？

Reply:
意図はないので修正します。

Reply:
FYI

https://developer.hashicorp.com/terraform/language/files/dependency-lock

> Terraform automatically creates or updates the dependency lock file each time you run [the terraform init command](https://developer.hashicorp.com/terraform/cli/commands/init). You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.